libs/common/src/assets/i18n/ca.json

```diff
@@ -95,11 +95,13 @@
 	"accounts.no_users_found": "no existeix cap usuari amb aquest correu",
 	"accounts.users": "usuari/s",
 	"activity.actor": "Actor",
+	"activity.chat": "Preguntes i respostes generatives",
 	"activity.load-more": "TODO",
```

Comment:
I see you didn't add it, but is this `load-more` used? if so you should replace the TODO by the proper translation and if not you can delete this entry

Reply:
fixed